README.md

```diff
@@ -4,6 +4,9 @@ NEAR Lake Framework is a small library companion to [NEAR Lake](https://github.c
 your own indexer that subscribes to the stream of blocks from the NEAR Lake data source and create your own logic to process
 the NEAR Protocol data.
 
+> [Official NEAR Lake Framework launch announcement](https://gov.near.org/t/announcement-near-lake-framework-brand-new-word-in-indexer-building-approach/17668) has been published on the NEAR Gov Forum
+> Greetings from the Data Platform Team! We are happy and proud to announce an MVP release of a brand new word in indexer building approach - NEAR Lake Framework.
```

Comment:
It is not readable:

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/304265/161034697-d2c5e3a1-d725-4c91-989c-9690c4762906.png">

Reply:
Ok, I do like the blockquote tag, but it's not so attractive in the dark mode :(

Reply:
I should have been more explicit. The fact that two paragraphs were joined into a single one without even a dot after "Forum" before "Greetings" makes it a confusing sentence. I am fine with the blockquote tag